pycbc/inference/models/brute_marg.py

```diff
@@ -20,7 +20,7 @@
 import logging
 import numpy
 
-from multiprocessing import Pool
```

Comment:
Possibly stupid question - some places have multiprocessing.Pool, other have multiprocessing.pool.Pool. This is how it was before these changes as well, so it must be okay, but wanted to flag it

Reply:
If using 'multiprocessing.Pool' I believe this will use default context as defined by set_start_context. In this case one can use 'get_context("fork/spawn").Pool()'